docs/docs/axes/cartesian/time.md

```diff
@@ -6,7 +6,7 @@ The time scale is used to display times and dates. Data are spread according to
 
 ## Date Adapters
 
-The time scale **requires** both a date library and corresponding adapter to be present. Please choose from the [available adapters](https://github.com/chartjs/awesome#adapters).
+The time scale **requires** both a date library and a corresponding adapter to be present. Please choose from the [available adapters](https://github.com/chartjs/awesome#adapters).
```

Comment:
Shoulnd't that be `an corresponding adapter`?

Reply:
@kurkle if your suggested change would be `The time scale requires both a date library and an corresponding adapter`, then I would say that doesn't work grammatically. Per [this article](https://www.grammar.com/a-vs-an-when-to-use/).

Reply:
Ok, looks like I missed a class some decades ago :)